src/utils/VisibilitySingleton.ts

```
@@ -1,5 +1,5 @@
 import { ChromeAPI } from '@redhat-cloud-services/types';
-import { isProd } from './common';
+import { isProd, LOGIN_SCOPES_STORAGE_KEY } from './common';
```

ESLint

[error] Member 'LOGIN_SCOPES_STORAGE_KEY' of the import declaration should be sorted alphabetically.

(sort-imports)